js/src/Routes.tsx

```diff
@@ -1,9 +1,9 @@
 import { AmandaPage } from './personal/amanda/Amanda.page.tsx'
 import { BryanPage } from './personal/bryan/Bryan.page.tsx'
-import { MainPage as BryanMainPage } from './personal/bryan/Main.page.tsx'
-import { ProjectsPage as BryanProjectsPage } from './personal/bryan/Projects.page.tsx'
-import { FunPage as BryanFunPage } from './personal/bryan/Fun.page.tsx'
-import { AboutMePage as BryanAboutMePage } from './personal/bryan/AboutMe.page.tsx'
+import { MainPage as BryanMainPage } from './personal/bryan/Main/Main.page.tsx'
+import { ProjectsPage as BryanProjectsPage } from './personal/bryan/Projects/Projects.page.tsx'
+import { FunPage as BryanFunPage } from './personal/bryan/Fun/Fun.page.tsx'
+import { AboutMePage as BryanAboutMePage } from './personal/bryan/About Me/AboutMe.page.tsx'
```

Comment:
nit: don't have spaces in directory and file names. Also not sure if Henry suggested any convention on casing? The mix of starting with lower and upper case is a bit weird